examples/chip-tool/templates/tests/tests.js

```diff
@@ -92,7 +92,6 @@ function getManualTests()
   ];
 
   const GeneralCommissioning = [
-    'Test_TC_GC_1_1',
```

Comment:
Need to update `examples/chip-tool-darwin/templates/tests.js` too.

Reply:
for Darwin i updated in " src/darwin/Framework/CHIP/templates/tests/tests.js " is this required

Reply:
You need to update both for now.  We're working on getting rid of `src/darwin/Framework/CHIP/templates/tests/tests.js` and only having the chip-tool-darwin bit, but we're not there yet.

Reply:
ok